benchopt/cli/tests/test_cli.py

```diff
@@ -625,13 +635,16 @@ def get_result(self): pass
         solver = missing_deps_cls.format(Cls='Solver')
         with temp_benchmark(solvers=[solver]) as benchmark:
             match = "not importable:\nSolver\n- buggy-class"
-            with pytest.raises(AttributeError, match=match):
+            with pytest.raises(AttributeError, match=re.escape(match)):
                 with CaptureRunOutput():
                     install([
                         *f'{benchmark.benchmark_dir} -s buggy-class -y '
                         f'--env-name {test_env_name}'.split()
                     ], 'benchopt', standalone_mode=False)
 
+    @pytest.mark.skipif(sys.platform == 'win32',
+                        reason="Skipping test on Windows because "
+                        "of --skip-install")
```

Comment:
Not sure I understand why this is needed?

Reply:
I skipped this test because I couldn't find any solution to the errors it displayed.

`test_no_error_minimal_requirements`  only fails when the `--skip-install` parameter is added onto `pytest`. 
 (here's the [error message](https://github.com/benchopt/benchopt/actions/runs/9679505377/job/26705655670?pr=717#step:10:2179))

I also couldn't solve the errors of 3 other tests which you'll find I skipped : 

- `test_solver_install` : does not work for the `R-PGD` solver as it needs R to be added manually to `Path` (cf. this [link](https://stackoverflow.com/questions/63863449/oserror-cannot-load-library-c-program-files-r-r-4-0-2-bin-x64-r-dll-error-0))
- `test_config_file_permission_no_warning` : displays a warning on the permission of the temporary file. I believe permission modification is more restricted in Windows (cf. this [link](https://stackoverflow.com/questions/72944021/pythons-os-chmod-behaves-differently-on-linux-vs-windows))
- `test_benchopt_install_in_env_with_requirements` : No clear idea yet why this fails. (Here's the [error message](https://github.com/benchopt/benchopt/actions/runs/9679242536/job/26704802386?pr=717#step:10:1201))

I believe the first 2 tests should be skipped, as making them work may need some more in-depth modifications of the code and they don't seem vital for the proper functioning of Benchopt. However, I am unsure about the remaining two. I've skipped all four tests for the time being, awaiting your feedback.

Reply:
I agree for the first two tests, I tried to push a fix for the second ones. Let's see the debug log :)